cbdr/cortex
# bugfix/COR-687: Fix Sizing of Media Library Popup

**Purpose:** To get rid of horizontal scrolling and overflow for media library popups on small screens. 

**JIRA:** [/browse/COR-687](https://cb-content-enablement.atlassian.net/browse/COR-687)


**Changes:**
* Changes to setup
  * n/a

* Architectural changes
  * n/a

* Migrations
  * n/a
  
* Library changes
  * n/a

* Side effects
  * n/a

**Screenshots**
* Before
![cortex screenshot](https://cloud.githubusercontent.com/assets/10361180/24717786/4009db16-19f9-11e7-9628-066a5cd22d9b.png)

* After

![cortex administration employer blogs new 2017-04-05 12-11-18](https://cloud.githubusercontent.com/assets/10361180/24717794/44f3bbce-19f9-11e7-88c2-abc3e3ad10cf.png)

**QA Links:**
[http://web.cortex-1.development.c66.me/](http://web.cortex-1.development.c66.me/)

**How to Verify These Changes**

* Steps to take
  * go to [http://web.cortex-1.development.c66.me/](http://web.cortex-1.development.c66.me/) try and make a blog post and insert media. Reduce the browser window size to see if the popup overflows of gets misaligned. 

* Responsive considerations
  * yes it should scale with width


**Relevant PRs/Dependencies:**

n/a
**Additional Information**
n/a
